hushline/routes.py

```diff
@@ -96,34 +96,30 @@ def inbox() -> Response | str:
 
         logged_in_user_id = session["user_id"]
         requested_username = request.args.get("username")
-        logged_in_user = User.query.get(logged_in_user_id)
-        if logged_in_user:
-            logged_in_username = logged_in_user.primary_username
-
-        if requested_username and requested_username != logged_in_username:
+        logged_in_user = db.session.get(User, logged_in_user_id)
+        if not logged_in_user or requested_username != logged_in_user.primary_username:
             return redirect(url_for("inbox"))
 
-        primary_user = User.query.get(logged_in_user_id)
-        if primary_user:
-            messages = (
-                Message.query.filter_by(user_id=primary_user.id).order_by(Message.id.desc()).all()
-            )
-            secondary_users_dict = {su.id: su for su in primary_user.secondary_usernames}
+        if logged_in_user:
+            messages = db.session.scalars(
+                db.select(Message).filter_by(user_id=logged_in_user.id).order_by(Message.id.desc())
+            ).all()
+            secondary_users_dict = {su.id: su for su in list(logged_in_user.secondary_usernames)}
```

Comment:
This line needs extra eyes on it. MyPy didn't like iterating over the `secondary_usernames` type, I _think_ create a list  from it should resolve the lazy references.

Reply:
This is no longer an issue after adopting the new declarative table definition